dashboards/component/chart/serializers.py

```diff
@@ -182,12 +183,13 @@ class Media:
 
 
 class ChartSerializer(ModelDataMixin, PlotlyChartSerializer):
-    """
-    Default chart serializer to read data from a django model
-    and serialize it to something plotly js can render
-    """
-
     class Meta(ModelDataMixin.Meta, PlotlyChartSerializer.Meta):
-        pass
+        filter_component = Filter  # Added the Filter component here
```

Comment:
I don't think we want to set a default here, it should just be a type annotation and default to `None` as many use cases wont want a filter on a chart

Reply:
It should also be the `FilterSet` that gets passed into the filter component rather than the filter component itself